cronsetup/cronsetup.go

```diff
@@ -0,0 +1,62 @@
+package cronsetup
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Scalingo/go-etcd-cron"
+	"github.com/Scalingo/go-utils/logger"
+	"github.com/coreos/etcd/clientv3"
+)
+
+type SetupOpts struct {
+	EtcdConfig func() (clientv3.Config, error)
+	Jobs       []etcdcron.Job
```

Comment:
Shouldn't you import `etcdcron`?

Reply:
the package is imported by `+	"github.com/Scalingo/go-etcd-cron"`

Reply:
Shouldn't you import it with `etcdcron "github.com/Scalingo/go-etcd-cron"` so that it works?